robotbase/src/main/java/com/team2357/frc2022/commands/ShootAgainstHubCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.team2357.frc2022.commands;
```

Comment:
You have the right idea, but this should be a command group that uses the already existing speed commands. Also, the shooter motors will have to start before the feeder/kicker motors, so that the shooter can spin up to the set speed.

Reply:
So should I do some sort of delay after starting the shooter motors or is there a better way to do it?

Reply:
A delay is fine.